spec.html

```diff
@@ -37285,9 +37403,9 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
         1. Let _jsonString_ be ? ToString(_text_).
         1. [id="step-json-parse-validate"] Parse ! StringToCodePoints(_jsonString_) as a JSON text as specified in ECMA-404. Throw a *SyntaxError* exception if it is not a valid JSON text as defined in that specification.
         1. Let _scriptString_ be the string-concatenation of *"("*, _jsonString_, and *");"*.
-        1. Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|).
+        1. [id="step-json-parse-parse"] Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|).
```

Comment:
```suggestion
        1. [id="step-json-parse-parse"] Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|). The extended early error rules defined in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref> must not be applied.
```

Reply:
This is probably less of a concern, but I think it might be better if this one made a similar change, something like:
> Note that one of the early error rules defined in [xref] is specified to not apply in this invocation of ParseText.

This too could just be a NOTE step.

Reply:
```suggestion
        1. [id="step-json-parse-parse"] Let _script_ be ParseText(! StringToCodePoints(_scriptString_), |Script|).
        1. NOTE: The early error rules defined in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref> have special handling for the above invocation of ParseText.
```